src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -801,7 +833,60 @@ private void renameGitHubRepo(Channel channel, User sender, String repo, String
             e.printStackTrace();
         }
     }
-    
+
+    /**
+     * @param repoName
+     *      Name of repo
+     */
+    boolean addGithubLabels(Channel channel, User sender, String repoName) {
+        boolean result = false;
+        OutputChannel out = channel.send();
+        try {
+            if (!isSenderAuthorized(channel,sender)) {
+                insufficientPermissionError(channel);
+                return false;
+            }
+
+            GitHub github = GitHub.connect();
+            GHOrganization org = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
+            GHRepository repo = org.getRepository(repoName);
+            if(repo == null) {
+                out.message("Repository with name "+repoName+" does not exist "+IrcBotConfig.GITHUB_ORGANIZATION);
+                return false;
+            }
+            Map<String, GHLabel> ghLabels = repo.listLabels().asList().stream().collect(Collectors.toMap(GHLabel::getName, l -> l));
+            out.message("Setting labels for "+repoName);
+            for (RepoLabels.Label label : labelsYAML.labels) {
+                if (!Strings.isNullOrEmpty(label.oldname) && ghLabels.containsKey(label.oldname)) { // ex fix => bugfix
+                    if (ghLabels.containsKey(label.name)) {
+                        out.message("Old label of " + label.oldname + " still exists, so can't rename to " + label.name);
+                        continue;
+                    }
+                } else if (ghLabels.containsKey(label.name)) {
+                    /*
+                     * when updateLabel is published
+                     * repo.updateLabel(
+                     *  Strings.isNullOrEmpty(label.oldname) ? label.name : label.oldname,
+                     *  label.name, // newname
+                     *  label.color,
+                     *  label.description
+                     * )
+                     */
```

Comment:
From https://github.com/github-api/github-api/pull/724 :
```suggestion
                     ghLabels.get(Strings.isNullOrEmpty(label.oldname) ? label.name : label.oldname)
                         .update()
                         .name(label.name) // newname
                         .color(label.color)
                         .description(label.description)
                         .done();
```